graph/graph.go

```diff
@@ -41,6 +50,8 @@ func (g *Graph) LoadFile(path string) (*build.File, error) {
 	f, err := g.loadFile(path)
 	if err == nil {
 		g.files[path] = f
+	} else {
```

Comment:
I _think_ this is right? We can end up with a nil pointer dereference further down if f is nil.